### DISCUSSION TITLE
images and stl not showing ?

### DISCUSSION THREAD

**Discussion by colly05** [community/community#32057] [Other Feature Feedback, Questions, & Ideas]
new user just browsing and i can not seam to fix this issue i get this 
Unable to render code block
my laptop is fine but the main pc no what am i missing 

https://github.com/flyingsonu122/skyline/blob/main/flyingsonu122-2020.stl
![image](https://user-images.githubusercontent.com/85044944/188590492-41d964b3-35a9-4190-a7da-bb0fee0a2b39.png)

**Comment by DanielRibeiroMaciel-TomTom** [comment]
In my case it is `png` files.

**Comment by jfullstackdev** [comment]
that loaded fine right away on my end, even in Incognito 

![image](https://user-images.githubusercontent.com/47092464/190186031-8e7e3659-9b70-487b-bbd5-885203b9747f.png)

your computer might have low memory or the processor is not that fast or it's running heavy apps, or maybe your Internet connection, or extensions installed in the same browser profile

**Comment by colly05** [comment]
Mine was blocked in the hosts file (think that’s what’s it called) at work
so can’t view it

All working now
